galaxy-requirements.yml

```diff
@@ -13,6 +13,8 @@ roles:
     version: 3.1.1
   - src: geerlingguy.jenkins
     version: 4.3.0
+  - src: geerlingguy.java
```

Comment:
This geerlingguy.java role does not do much. It does contain a bunch of if then else for different Linux distros, but as we do not support other distros in any of the other roles. This looks to me like an unnecessary dependency for just
```
- name: Ensure Java is installed.
  package:
    name: "{{ java_packages }}"
    state: present
```
In addition it creates ```/etc/profile.d/java_home.sh```, which only contains
```
export JAVA_HOME={{ java_home }}
```
and I'm wondering if this is something that should be configured globally for every software package and every user on the system. I think it is safer to leave the default as is and tell Jenkins explicitly to use the Java version that is installed by our jenkins role.